src/spikeinterface/preprocessing/common_reference.py

```diff
@@ -34,29 +56,43 @@ class CommonReferenceRecording(BasePreprocessor):
         list of channels to be applied to each group is expected. If "single" reference, a list of one channel  or an
         int is expected.
     local_radius: tuple(int, int)
-        Use in the local CAR implementation as the selecting annulus (exclude radius, include radius)
-    verbose: bool
-        If True, output is verbose
+        Use in the local CAR implementation as the selecting annulus with the following format:
+
+        `(exclude radius, include radius)`
+
+        Where the exlude radius is the inner radius of the annulus and the include radius is the outer radius of the
+        annulus. The exclude radius is used to exclude channels that are too close to the reference channel and the
+        include radius delintes the outer boundary of the annulus whose role is to exclude channels
+        that are too far away.
+
     dtype: None or dtype
         If None the parent dtype is kept.
 
     Returns
     -------
     referenced_recording: CommonReferenceRecording
         The re-referenced recording extractor object
+
+
+    Notes
+    -----
+        Note that the references are calculated within a `get_traces` context. That is, the reference is calculated
+    for each chunk of traces that is requested. Which means that `get_traces(stat_frame=0, end_frame=10_000)`
+    will be different from `get_traces(start_frame=0, end_frame=20_000)[0:10_000]`. While in most cases the estimation
+    of the mean or median will be the same, it is important to keep this in mind when using this with small chunks.
```

Comment:
@h-mayorquin I think this is not true. The median/mean etc. is calculated for each sample separately so it doesn't matter how you retrieve the traces

Reply:
Yes, thanks, the notes are wrong. The average is across the channels not across time right? So it will be affected if you slice channels but I think that right now you slice over all the channels at the beginning to avoid this. 

Thanks!